### ISSUE TITLE
Unable to represent a very small and a very large value together in a bar graph.

### ISSUE THREAD

**Issue by jthakkar28** [jtblin/angular-chart.js#693]
Example:  I want to represent two values 0.1 and 90 in the same bar graph, 90 is visually visible whereas 0.1 is not visible, I see the value 0.1 only after hovering, I want to show smallest possible visual representation for such values, just so the user knows there is some value.

**Comment by jtblin** [comment]
This is a usage question or stale support request from several years ago. As we are currently modernizing the library to support Chart.js v4 and TypeScript, we are closing stale issues. If this is still a problem in the latest version, please open a new issue with a modern reproduction.
